charts/cloudtty/_crds/cloudshell.cloudtty.io_cloudshells.yaml

```diff
@@ -113,7 +113,17 @@ spec:
                 type: string
               runAsUser:
                 type: string
+              secretRef:
+                description: 'SecretRef represents the secret contains mandatory credentials
```

Comment:
https://github.com/cloudtty/cloudtty/blob/32343d294f9a3c16717b8dd261606c54b88a37bc/charts/cloudtty/_crds/cloudshell.cloudtty.io_cloudshells.yaml#L69

When using configmap, it is directly kube config, which is inconsistent with here, is this suitable?

Reply:
Should we describe the priority of configmap and secretRef?

Reply:
These two cannot be used together.

If open AllowSecretStoreKubeconfig, we will directly use the secret to restore kubeconfig. Currently AllowSecretStoreKubeconfig is false by default, we can still use the configMap.

Reply:
yes, I mean, we should write it down to the description.

Reply:
ok, thanks